samples/boards/mec15xxevb_assy6853/power_management/sample.yaml

```diff
@@ -2,5 +2,5 @@ sample:
   name: MEC150x board sample power management tests
 tests:
   sample.board.mec15xxevb_assy6853:
-    platform_allow: mec15xxevb_assy6853
+    skip: True
```

Comment:
skip is wrong here, this is a sample that can be run, so let it run and capture the output and check if it matches what is expected.

Reply:
OK， I‘ll check it and re-push later.